funnel/templates/project_cfp.html.jinja2

```diff
@@ -26,8 +26,8 @@
     {%- endfor %}
     {% if not form.cfp_end_at.data %}
       <div id="cfp-publish-actions" class="btn-wrapper hide">
-        <button name="open-now" class="mui-btn cfp-action-btn">{% trans %}Open CfP now{% endtrans %}</button>
-        <button name="later" class="mui-btn cfp-action-btn">{% trans %}Open CfP later{% endtrans %}</button>
+        <button name="open-now" class="mui-btn cfp-action-btn">{% trans %}Open proposal submissions now{% endtrans %}</button>
+        <button name="later" class="mui-btn cfp-action-btn">{% trans %}Open proposal submissions later{% endtrans %}</button>
```

Comment:
These two buttons side by side will be a mouthful. Can't these be "Open now" and "Open later"? As long as something else clarifies what is being opened.

Also, "open proposal submissions now" could be misinterpreted as a link to see existing submissions.